src/scroll-timeline-base.js

```diff
@@ -492,11 +492,11 @@ export class ScrollTimeline {
 
     // Step 3
     if (currentScrollOffset < startOffset)
-      return CSS.percent(0);
+      return CSS.percent(-0.001);
```

Comment:
Leave at 0 and 100% for this PR.  We can discuss how to handle before and after phase separately.  My preference is to eliminate the clamping entirely and rework scrollOfffsets.